### PR TITLE
Feature/erssup 76095

### PR DESCRIPTION
**Pull request by nhsd-jack-wainwright**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: proxies/live/apiproxy/targets/ers-target.xml [hunk 1]
```diff
@@ -1,26 +1,46 @@
...
+                <Condition>faultstring ~ "*OauthV2.VerifyAccessToken.scopeSet*"</Condition>
+            </Step>
             <Step>
                 <Name>AssignMessage.OAuthPolicyErrorResponse</Name>
+                <!--Condition is implemented this way around to account for aalError being null (https://docs.apigee.com/api-platform/reference/conditions-reference#behaviorofnulloperandsinconditionalstatements)-->
+                <Condition>aalError != true</Condition>
+            </Step>
+            <Step>
+                <Name>AssignMessage.AuthenticationOperationOutcomeErrorResponse</Name>
+                <!--Condition is implemented this way around to account for aalError being null (https://docs.apigee.com/api-platform/reference/conditions-reference#behaviorofnulloperandsinconditionalstatements)-->
```
Comment (nhsd-jack-wainwright) [inline proxies/live/apiproxy/targets/ers-target.xml:42]: This comment should be against line 16

File: tests/data.py [hunk 2]
```diff
@@ -1,22 +1,90 @@
...
+    """
+
+    AAL3 = "aal3"
+    AAL2 = "aal2"
+    AAL1 = "aal1"
 
 
 class Actor(Enum):
     # User ID, org code, business function, IAL, OBO User ID
-    RC = ("555032000100", "D82106", "REFERRING_CLINICIAN", "3")
```
Comment (nhsd-david-wass) [inline tests/data.py:6]: Is this comment correct for the order of the parameters?

**Comment by nhsd-david-wass** [comment]
Looks good - just a minor comment about a comment

File: tests/data.py (hunk 2 above)
Comment (nhsd-jack-wainwright) [reply tests/data.py:6]: Nope well spotted, I'll update these to properly match the arguments.

**Review by nhsd-david-wass — DISMISSED** [review]

**Review by nhsd-david-wass — APPROVED** [review]
reapproving for merge

**Pull request merged**